MyPy linting errors

It seems we need to take a look at the issues below:

```
mypy.....................................................................Failed

test/test_template_v2.py:665: error: "TemplateV2" has no attribute "setUp"  [attr-defined]
test/test_template_v2.py:666: error: "TemplateV2" has no attribute "test_temp_score"  [attr-defined]
test/test_template_v2.py:667: error: "TemplateV2" has no attribute "test_target_projections"  [attr-defined]
test/test_template_v2.py:668: error: "TemplateV2" has no attribute "test_get_company_data"; maybe "template_company_data"?  [attr-defined]
test/test_template_v2.py:671: error: "TemplateV2" has no attribute "setUp"  [attr-defined]
```

--- a/test/test_template_v2.py
+++ b/test/test_template_v2.py
@@ -1,5 +1,6 @@
 import json
 import os
+import sys
 from typing import List
 from unittest import TestCase
 
@@ -661,11 +662,4 @@ def test_ch4_gwp(template_V2_S3: TemplateV2):
 
 
 if __name__ == "__main__":
-    test_PC = template_V2_PC()
-    test_PC.setUp()
-    test_PC.test_temp_score()
-    test_PC.test_target_projections()
-    test_PC.test_get_company_data()
-
-    test_S3 = template_V2_S3()
-    test_S3.setUp()
+    sys.exit(pytest.main())

